sdk/jobs/pipelines/3a_pipeline_with_spark/spark-in-pipeline.ipynb

```diff
@@ -0,0 +1,119 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import required libraries\n",
+    "from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential\n",
+    "from azure.ml import MLClient, Input, Output\n",
+    "from azure.ml import command, spark\n",
+    "from azure.ml.dsl import pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    credential = DefaultAzureCredential()\n",
+    "    # Check if given credential can get token successfully.\n",
+    "    credential.get_token(\"https://management.azure.com/.default\")\n",
+    "except Exception as ex:\n",
+    "    # Fall back to InteractiveBrowserCredential in case DefaultAzureCredential not work\n",
+    "    credential = InteractiveBrowserCredential()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get a handle to workspace\n",
+    "ml_client = MLClient.from_config(credential=credential)\n",
+    "\n",
+    "# Retrieve an already attached Azure Machine Learning Compute.\n",
+    "synapse_compute = \"synapse-compute\"\n",
+    "print(ml_client.compute.get(synapse_compute))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define pipeline\n",
+    "@pipeline(description=\"Spark pipeline job\",)\n",
```

Comment:
Maybe add another case that user define spark component via yaml and load_component()?
This is the authoring way that we see most users going now

Reply:
sure but question: does that experience depend on the specific job type or it that identical across spark/non-spark jobs?

Reply:
Done. But I only could import it like so: `from azure.ml.entities import load_component`, which is different from how it's shown [here](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-create-component-pipeline-python#import-required-libraries) - that module doesn't exist! Can you confirm 3p can use `azure.ml.entities` directly? Names don't have `_` so I assume yes but let's confirm.

Reply:
load_component is generic for all type of jobs/components.

I believe below should work:
from azure.ai.ml import load_component

Reply:
Same as my comment above - I can't import from `azure.ai.ml`. See screenshot below. Can you confirm that works on your side? In my setup, I have a virtualenv with SDK/CLIv2 installed via `dev_setup.py -sc` script. 

![image](https://user-images.githubusercontent.com/38863503/180693479-aa9bcb91-c582-4ea3-af4e-77d40515a347.png)